services/movie_session.py

```diff
@@ -42,3 +42,8 @@
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_taken_seats(movie_session_id: MovieSession) -> QuerySet:
+    tickets = Ticket.objects.filter(movie_session_id=movie_session_id)
+    return [{"row: ticket.row, seat: ticket.seat"} for ticket in tickets]
```

Comment:
There is a syntax error in the list comprehension. The dictionary should be constructed with keys as strings: `{'row': ticket.row, 'seat': ticket.seat}` instead of `{"row: ticket.row, seat: ticket.seat"}`.